### PR TITLE
Clarify path for migrating from 5.x

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | 1.13+

Clarifies that migration from Publish Platform cannot be done instantly, but has to go through 1.7 and 1.13 as intermediary steps.

### CODE REVIEW

File: docs/migrating/migrating_from_ez_publish_platform.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+eZ Publish Platform (5.x) was a transitional version of the eZ CMS, bridging the gap between the earlier generation called eZ Publish (sometimes referred to as *legacy*), and the current eZ Platform and eZ Platform Enterprise Edition for developers.
 
-eZ Publish Platform introduced a new Symfony-based technology stack that could be run along the old (*legacy*) one. This bridging is still possible using something called Legacy Bridge, an optional package for eZ Platform. This fluid change allows eZ Publish users to migrate to eZ Platform in two steps, using the bridging as an intermediary stepping stone.
+eZ Publish Platform introduced a new Symfony-based technology stack that could be run along the old (*legacy*) one. This bridging is still possible using something called Legacy Bridge, an optional package for eZ Platform. This fluid change allows eZ Publish users to migrate to eZ Platform in steps, using the bridging as an intermediary stepping stone.
```
Comment (MagdalenaZuba) [inline docs/migrating/migrating_from_ez_publish_platform.md]: migrate to `eZ Platform gradually, using`
OR
migrate to eZ Platform `in few steps`, using

File: docs/migrating/migrating_from_ez_publish_platform.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
 
-## Upgrading eZ Publish Platform 5.4.x (Enterprise-) / 2014.11 (Community-edition) to eZ Platform v1.11 or higher
+## Upgrade process
+
+An upgrade from eZ Publish Platform 5.4.x (Enterprise edition) or 2014.11 (Community edition) to newer versions of eZ Platform must be performed in steps.
```
Comment (MagdalenaZuba) [inline docs/migrating/migrating_from_ez_publish_platform.md]: performed gradually.
OR
performed incrementally.
OR
in stages.

File: docs/migrating/migrating_from_ez_publish_platform.md [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
 
-eZ Publish Platform introduced a new Symfony-based technology stack that could be run along the old (*legacy*) one. This bridging is still possible using something called Legacy Bridge, an optional package for eZ Platform. This fluid change allows eZ Publish users to migrate to eZ Platform in two steps, using the bridging as an intermediary stepping stone.
+eZ Publish Platform introduced a new Symfony-based technology stack that could be run along the old (*legacy*) one. This bridging is still possible using something called Legacy Bridge, an optional package for eZ Platform. This fluid change allows eZ Publish users to migrate to eZ Platform gradually, using the bridging as an intermediary stepping stone.
 
-## Upgrading eZ Publish Platform 5.4.x (Enterprise-) / 2014.11 (Community-edition) to eZ Platform v1.11 or higher
+## Upgrade process
+
+An upgrade from eZ Publish Platform 5.4.x (Enterprise edition) or 2014.11 (Community edition) to newer versions of eZ Platform must be performed in stages.
+
+You can upgrade from eZ Publish Platform directly to the v1.7 LTS release.
```
[the remark this answers is not in the document]
Comment (kmadejski) [reply docs/migrating/migrating_from_ez_publish_platform.md]: > hmm where did this come from?

I asked for this documentation clarification. I noticed an issue while working on customer's migration problems.

`ezflow-migration-toolkit` depends on `EzSystems\LandingPageFieldTypeBundle\FieldType\LandingPage\Model\Block\Item` (ref: https://github.com/ezsystems/ezflow-migration-toolkit/blob/master/lib/Mapper/BlockMapper.php#L5) which has been removed in `LandingPageFieldTypeBundle` 1.10, therefore it's not possible to run migration in v1.13.

[the remark this answers is not in the document]
Comment (andrerom) [reply docs/migrating/migrating_from_ez_publish_platform.md]: ok, but then we should rather fix that, as it's then broken.

**Pull request merged**